data_platform/ingestion/data_dumps/bluesky/athena.py

```
@@ -0,0 +1,144 @@
+def _wait_for_query_completion(client: Any, execution_id: str) -> str:
+    while True:
+        state = client.get_query_execution(QueryExecutionId=execution_id)
+        status = state["QueryExecution"]["Status"]["State"]
+        if status == QUERY_STATUS_SUCCEEDED:
+            return execution_id
+        if status in TERMINAL_FAILURE_STATES:
+            reason = state["QueryExecution"]["Status"].get(
+                "StateChangeReason",
+                UNKNOWN_STATE_CHANGE_REASON,
+            )
+            raise RuntimeError(f"Athena query {status}: {reason}")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)